docs/core/tutorials/aspnet-core.md

```diff
@@ -7,4 +7,4 @@ ms.date: 06/20/2016
 ---
 # Getting started with ASP.NET Core
 
-For tutorials about developing ASP.NET Core web applications, we suggest you head over to [ASP.NET Core documentation](/aspnet/core/).
+For tutorials about developing ASP.NET Core web applications, tions, see [ASP.NET Core documentation](aspnet/core/tutorials).
```

Comment:
remove ",tions,"

also change the link text to ASP.NET Core tutorials since you changed the destination